docs/can-guides/upgrade/migrating-to-canjs-6.md

```diff
@@ -534,3 +534,17 @@ class MyParent extends StacheElement {
 customElements.define("my-parent", MyParent);
 ```
 @codepen
+
+
+### inserted/removed event
+Starting from CanJS 4, __inserted__ and __removed__ events are no longer exist.
+
+If you still need to listen to those events on an element, you have to write the following:
```

Comment:
A general tip for writing in English:  your writing will seem more formal if you can avoid using "you". Teachers drill this into us in school, and lower our grades for essays if we fail to follow it.

I'm not requesting a change here, since our guides are already full of this informality (and others), but suggesting how to improve future docs updates.

Reply:
Thank you for the tip, I appreciate it!